src/lib/tvgShape.cpp

```diff
@@ -205,7 +205,7 @@ Result Shape::appendArc(float cx, float cy, float radius, float startAngle, floa
     }
 
     if (pie) {
-        pImpl->path->moveTo(cx, cy);
```

Comment:
As reviewed, It looks neither moveTo nor lineTo not necessary because close() will automatically works as line To start point.
Could you please double-check if it works fine only when call close()?

Reply:
sure, it works fine.